emcc.py

```diff
@@ -2084,11 +2090,6 @@ def check_human_readable_list(items):
       if shared.Settings.DEBUG_LEVEL >= 3 and not shared.Settings.WASM and not shared.Settings.WASM_BACKEND:
         newargs.append('-fno-discard-value-names')
```

Comment:
this item will not be in `compile_args` now, but it was before. is that ok?

Reply:
Good point.  I cleaned this up a little more.